src/main/kotlin/no/skatteetaten/aurora/boober/service/OpenShiftObjectGenerator.kt

```diff
@@ -110,7 +112,11 @@ class OpenShiftObjectGenerator(
         val namespace = namespace {
             apiVersion = "v1"
             metadata {
-                labels = mapOf("affiliation" to environment.affiliation)
+                val ttl = environment.ttl?.let {
+                    val removeInstant = now + it
+                    "removeAfter" to removeInstant.toString()
```

Comment:
Blir formatet riktig?

Reply:
Formatet vil bli:
A string representation of this instant using ISO-8601 representation

Reply:
Som er det vi vil ha.